draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2321,7 +2346,7 @@ https://video.example.com/k/movie1/4/013.ts
              </list></t>
              <t>Property: media-protocol<list style="empty">
 
-                <t>Description: Media protocol to be when parsing and interpreting this playlist. </t>
+                <t>Description: Media protocol to be used when parsing and interpreting this playlist. </t>
```

Comment:
Kevin made the following point:

> does this need to be explicit?  can it not be inferred from either the file extension, mime type, or file header?

Since this can be inferred, should we remove this?

Reply:
lets not remove it for now and leave it for discussion.
one reason for keeping it is that the value is that it is also being refered to by a capability object